external-crates/move/crates/move-vm-config/src/verifier.rs

```diff
@@ -3,6 +3,7 @@
 
 pub const DEFAULT_MAX_CONSTANT_VECTOR_LEN: u64 = 1024 * 1024;
 pub const DEFAULT_MAX_IDENTIFIER_LENGTH: u64 = 128;
+pub const DEFAULT_MAX_VARIANTS: u64 = 127;
```

Comment:
I think adding some debug (or even compile-time) asserts would be good. I'd hesitate to make this configurable right now, since I don't want us to accidentally set this to more than 127 as that will then have pretty significant consequences around uleb encoding these later possibly.

Reply:
Added some compile-time asserts that this is 127.